client/src/views/Models/components/DrilldownPanel/KnowledgePane.vue

```diff
@@ -3,7 +3,7 @@
      <div v-if="!isEmptyData" class="mt-3 documents-container hide-scrollbar">
       <div class="mb-1 px-2 py-4 d-flex rounded-lg border" v-for="(object, index) in data.objects" :key="index">
         <div class="flex-grow-1">
-            <a :href="object.bibjson.link[0].url" target="_blank">
+            <a href="/" @click="e => openModal(e, index)">
```

Comment:
Because this is not a link anymore, you should use a `<button type="button" class="btn btn-link" />` instead.